database/migrate.sh

```
@@ -0,0 +1,222 @@
+    
+    ls -1 "$MIGRATIONS_DIR"/*.sql | sort -V | while read -r file; do
+        apply_migration "$file"
+    done
+    
+    log "All migrations completed successfully!"
+}
+
+# Function to rollback migration
+rollback_migration() {
+    local migration_number="$1"
+    local migration_files=("$MIGRATIONS_DIR"/${migration_number}_*.sql)
```

Shellcheck (0.11.0)

[warning] Quote to prevent word splitting/globbing, or split robustly with mapfile or read -a.

(SC2206)